### ISSUE TITLE
Has this been tested?

### ISSUE THREAD

**Issue by john--** [aramaerospace/Micro-VESC-6#1]
Looks great. Just curious if it's been tested?

**Comment by aramaerospace** [comment]
Yes, it has been tested, worked fine. Here is a picture of an assembled board.

![Controller_1](https://user-images.githubusercontent.com/13492691/63664478-894de880-c77b-11e9-8b3b-4069102261eb.jpg)

**Comment by john--** [comment]
Wow, great work! 
I'm tempted to build this thing. Where did you order your components?

**Comment by john--** [comment]
@aramaerospace sorry for the delay, that would actually be great! I had the PCB made at JLCPCB, my goto place. Shipping will take some time though, so I'm in rush to get my BOM ordered.

**Comment by aramaerospace** [comment]
Added the BOM, hope this helps. I've also corrected some issues on the PCB: some of the connections from DRV8301 to the FETs where messed up. It is corrected now. If you've ordered your boards, I'd advice to re-order them. Sorry for that!

**Comment by john--** [comment]
Oh nice, thanks! Oh don't worry about it, PCBs aren't terribly expensive. Really appreciate your work here!
